RecoMuon/L2MuonIsolationProducer/python/hltL2MuonIsolationsCR_cfi.py

```diff
@@ -1,77 +1,49 @@
 import FWCore.ParameterSet.Config as cms
+import RecoMuon.L2MuonIsolationProducer.hltL2MuonIsolations_cfi as _mod
```

Comment:
I thought the preferred way of doing this was
```suggestion
from RecoMuon.L2MuonIsolationProducer.hltL2MuonIsolations_cfi import hltL2MuonIsolations as _hltL2MuonIsolations
```
and later
```python
hltL2MuonIsolationsCR = _hltL2MuonIsolations.clone(...)
```
?

I do not see a problem with this approach - but I would ask @cms-sw/reconstruction-l2 what style should be used, for better consistency ?

Reply:
@fwyzard I agree that for consistencies one could have assigned some more meaningful name for the temporary config
Since it is indeed a temporary, we agreed that a generic name as `_mod` (and reciclable in all PRs of the ongoing campaign) can be used instead

Reply:
@perrotta sorry, I guess I wasn't very clear. I didn't mean the temporary name, but whether the temporary should be `EDProducer`'s python instance, or the whole cfi module.
I.e.
```python
import Subsystem.Package.Module_cfi as _mod
module = _mod.module.clone(...)
```
vs
```python
from Subsystem.Package.Module_cfi import module as _module
module = _module.clone(...)
```

I do like the former, but so far I've always seen the latter being used.

Reply:
Ah, ok, sorry for not having realized what did you actually mean, and thank you for repeating.

In general, if one imports from complex python configs I agree that it is preferable to include the single module, instead of the whole file. In this case, as well as in most of the other ones dealt with by @jeongeun during this campaign, the imported config is the one automatically generated by the fillDescriptions method, and I personally don't see any drawback for doing it.

I'd say that if you have any reason I am not aware of for preferring including the module config instead of the whole file we can ask Jeongeun to do so: otherwise I'd stick on it, at least for this PR.